Makefile

```diff
@@ -43,7 +43,7 @@ gazelle:
 	$(bazelbin) run //:gazelle
 
 gazelle-update-repos:
```

Comment:
will this be removed in a follow up?

Reply:
yes, currently there's lane failure if you remove it. Therefore I just emptied it.